src/SFA.DAS.AssessorService.Web/Controllers/CertificateOptionController.cs

```diff
@@ -25,24 +25,24 @@ public CertificateOptionController(ILogger<CertificateController> logger, IHttpC
         }
 
         [HttpGet]
-        public async Task<IActionResult> Option(bool? redirectToCheck = false, bool? isFromStandard = false)
+        public async Task<IActionResult> Option(bool? redirectToCheck = false)
         {
-            var sessionString = SessionService.Get("CertificateSession");
+            var sessionString = SessionService.Get(nameof(CertificateSession));
             if (sessionString == null)
             {
                 return RedirectToAction("Index", "Search");
             }
-            
-            return await LoadViewModel("~/Views/Certificate/Option.cshtml", isFromStandard);
+
+            return await LoadViewModel("~/Views/Certificate/Option.cshtml");
         }
 
-        private async Task<IActionResult> LoadViewModel(string view, bool? isFromStandard)
+        private async Task<IActionResult> LoadViewModel(string view)
         {
             var username = ContextAccessor.HttpContext.User.FindFirst("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn")?.Value;
```

Comment:
This could be moved to the base class it is all over the place the same

Reply:
That was supposed to be line 41

Reply:
Have refactored into base class 👍